lib/aws-xray-sdk/facets/rack.rb

```diff
@@ -49,6 +49,8 @@ def call(env)
             resp_meta[:content_length] = len
           end
           segment.merge_http_response response: resp_meta
+          trace_header = {TRACE_HEADER => TraceHeader.from_entity(entity: XRay.recorder.current_segment).root_string}
```

Comment:
Nitpick. Why passing `XRay.recorder.current_segment` instead of `segment`?

Reply:
I think I have written `XRay.recorder.current_segment` too often to retrieve the current segment, that I didn't realize I actually have a segment object available in this scope. 🤔 Good catch and corrected! 🙂